osbrain/tests/common.py

```diff
@@ -6,6 +6,10 @@
 from osbrain.helper import sync_agent_logger
 
 
+def receive(agent, message, topic=None):
+    agent.received.append(message)
```

Comment:
Adding `_handler` is a bit too much for my taste, but I will change it to `append_received` 👍